tests/local_vespa/test_app/schemas/document_passage.sd

```diff
@@ -5,6 +5,15 @@ schema document_passage {
         stemming: none
     }
 
+    field language type string {
+        indexing: "en" | set_language
```

Comment:
Looking at the docs for this:

> The recommended use is to have one field in the document containing the language code, and that field should be the first field in the document, as it will only affect the fields defined after it in the schema.

https://docs.vespa.ai/en/reference/indexing-language-reference.html#set_language

This feels weird to me to have a document config at a field level, but it is what it is! I'm wondering if we need to move the language field above `text_block_not_stemmed`?

Reply:
yeah, this is weird, and very good catch! I've updated it and raised a feature request with Vespa to warn on deploy if this happens (I find the other warnings when deploying locally pretty useful) https://github.com/vespa-engine/vespa/issues/33053